UPGRADE-4.0.md

```diff
@@ -1,6 +1,12 @@
 UPGRADE FROM 3.x to 4.0
 =======================
 
+Debug
+-----
+
+ * `FlattenException::getTrace()` now returns additional type descriptions
+   `integer` and `float`.
+
```

Comment:
@xabbuh I am confused about the branching/release strategy and how changes such as this one are handled. I suppose the change will be included in Symfony 3.2 (and no previous versions), right? Which changelog file should I add it to? The newest CHANGELOG file in the master branch is for 3.1.